networks/ai85-facedet-tinierssd.yaml

```diff
@@ -0,0 +1,191 @@
+---
+# HWC (little data) configuration for FaceDetection
+# Parallel Model
```

Comment:
What does 'parallel model' stands for?

Reply:
Thank you for the comment. The 'Parallel Model' term comes from the original tinierssd model, and I believe it stands for the parallel outputs from different layers. I think we may remove it as it is not critical information.